.gas-snapshot

```diff
@@ -0,0 +1,10 @@
+APIConsumerIntegrationTest:test_apiRequestCallback() (gas: 98186)
```

Comment:
What's this file for?

Reply:
Ah that's an artefact from my local config running `forge snapshot` - useful to see diff in gas usage between changes when developing contracts but maybe not so relevant in this repo so will remove.